execution-engine/Cargo.toml

```diff
@@ -33,6 +33,7 @@ num-derive = { version = "0.3", default-features = false }
 num = { version = "0.4", default-features = false }
 anyhow = { version = "1.0", default-features = false }
 cfg-if = "1"
+log = "*"
```

Comment:
We should commit to a particular version of `log` here.

Reply:
It will be removed before merging, this is for debugging during dev.